vendor/vendor.json

```diff
@@ -369,5 +375,5 @@
 			"revisionTime": "2018-01-09T11:43:31Z"
 		}
 	],
-	"rootPath": "github.com/grafeas/grafeas"
+	"rootPath": "github.com/glassbearinc/grafeas"
```

Comment:
Still not right, please use github.com/grafeas/grafeas

Reply:
You caught this in the middle of me making that edit. Sorry!